kotlin-source/src/main/kotlin/com/example/api/ExampleApi.kt

```diff
@@ -75,22 +72,13 @@ class ExampleApi(val services: CordaRPCOps) {
      * The flow is invoked asynchronously. It returns a future when the flow's call() method returns.
      */
     @PUT
-    @Path("{party}/create-iou")
-    fun createIOU(iou: IOU, @PathParam("party") partyName: X500Name): Response {
-        val otherParty = services.partyFromX500Name(partyName)
-        if (otherParty == null) {
-            return Response.status(Response.Status.BAD_REQUEST).build()
-        }
-
-        val state = IOUState(
-                iou,
-                services.nodeIdentity().legalIdentity,
-                otherParty,
-                IOUContract())
+    @Path("create-iou")
+    fun createIOU(@QueryParam("iouValue") iouValue: Int, @QueryParam("partyName") partyName: X500Name): Response {
+        val otherParty = services.partyFromX500Name(partyName) ?:
+                return Response.status(Response.Status.BAD_REQUEST).build()
```

Comment:
when we catch an exception from the flow can we return more than "Transaction Failed"